### PR TITLE
Fusion 12 update

### PR DESCRIPTION
**Pull request by omonoukpoma**

### CODE REVIEW

**Comment by omonoukpoma** [comment]
this is on hold pending fusion-11-update merge

File: README.md [hunk 1]
```diff
@@ -15,3 +15,47 @@ As a user, I would like to be able to render many pieces of content via this tem
...
+As a user, I would like to group features together to create more complex layouts.
+
+## Guide
+To group features together, we need to create a chain. A [chain](https://redirector.arcpublishing.com/alc/arc-products/pagebuilder/user-docs/pagebuilder-editor-adding-features-and-chains/) is a collection of features organized in a predetermined structure. In this task, we will create two chains - a default chain and a grid chain.
+
+NOTE: All chains are places in the `components/chains` directory
+
+1. Create a new file called default.jsx in the `components/chains` directory
+
+2. In this file, create a functional component that all children within the props like so:
```
Comment (nicrocs) [inline README.md]: Is there a missing word here? `Renders` all children?

File: README.md [hunk 2]
```diff
@@ -15,3 +15,47 @@ As a user, I would like to be able to render many pieces of content via this tem
...
+    '12': '1',
+    '6': '2',
+    '4': '3',
+    '3': '4',
+  },
+  name: 'Number of Cards (Desktop)',
+}),
+```
+
+5. Iterate through the `children` props and each item in a div with the desktopCols, tabletCols and mobileCols values provided by the editor (using bootstrap)
```
Comment (nicrocs) [inline README.md]: Should this be something like `place each item in a div`

**Review by nicrocs — CHANGES_REQUESTED** [review]
Looks good. I think there are a couple of missing words, but otherwise was able to follow along no problem.

**Review by nicrocs — APPROVED** [review]
👍

**Pull request merged**